packages/babel-plugin-transform-object-rest-spread/README.md

```diff
@@ -20,6 +20,15 @@ $ npm install babel-plugin-transform-object-rest-spread
 }
 ```
 
+This plugin will polyfill `Object.assign` by default. If you want to prevent that use config this
```

Comment:
Hi @hzoo, thanks for your comment, I'm not exactly following though, how would you like it to read? Thanks :)